### ISSUE TITLE
Ensure duplicate landmark elements have unique labels

### ISSUE THREAD

**Issue by MelSumner** [ember-template-lint/ember-template-lint#1272]
If two of the same landmark elements or landmark roles are found in the same template, ensure that they have a unique label (provided by the `aria-label` or `aria-labelledby` attribute). 

## List of elements & their corresponding roles
- header (banner)
- main (main)
- aside (complementary)
- form (form, search)
- main (main)
- nav (navigation)
- footer (contentinfo)

### Allowed

```html
<nav aria-label="primary site navigation"></nav>
<nav aria-label="secondary site navigation within home page"></nav>
```

```html
<nav aria-label="primary site navigation"></nav>
<div role="navigation" aria-label="secondary site navigation within home page"></nav>
```

### Disallowed

```html
<nav></nav>
<nav></nav>
```

```html
<nav></nav>
<div role="navigation"></div>
```

```html
<nav aria-label="site navigation"></nav>
<nav aria-label="site navigation"></nav>
```

**Comment by rajasegar** [comment]
I am picking this one, @MelSumner @bmish Can you please assign it to me?
